src/index.ts

```diff
@@ -620,6 +657,7 @@ export async function init(config: IClientConfig): Promise<EppoJSClient> {
 /**
  * Used to access a singleton SDK client instance.
  * Use the method after calling init() to initialize the client.
+ * @deprecated
```

Comment:
consider adding what the user should do instead after `@deprecated` (e.g., why is this deprecated)